examples/6_totp.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+
+###################################
+# WHAT IS IN THIS EXAMPLE?
+#
+# This is a simple TOTP bot that makes use of
+# the team encrypted key-value store, which
```

Comment:
i expect for many people, this is the place where they will first be exposed to the existence of the key-value store. so, can we intro it here? oh also, i think maybe we should change the name of the file to something like `4_totp_simple_storage`? the number at the beginning was intended to be the complexity of the example. like, if you're learning about how keybase bot development works, start from the top and work your way down. i think we should probably intro this somewhere around the 4s (but i'm flexible). and totp_simple_storage... :man-shrugging:. but a little more deets are probably better. 

intro text. maybe something like...

Keybase has added an encrypted key-value store intended to support security-conscious bot development. It is a place to store small bits of data that are (1) encrypted for a team or user (via the user's implicit self-team: e.g. `alice,alice`), (2) persistent across logins, (3) fast and durable. It supports putting, getting, listing and deleting. There is also a concurrency primitive, but check out the other example for that. A team has many `namespaces`, a `namespace` has many `entryKeys`, and an `entryKey` has one current `entryValue`. Namespaces and EntryKeys are in cleartext, and the keybase client service will encrypt and sign the entryValue on the way in (as well as decrypt and verify on the way out) so keybase servers cannot see it or lie about it.